datafusion/core/tests/sql/expr.rs

```diff
@@ -510,6 +510,23 @@ async fn test_array_literals() -> Result<()> {
     Ok(())
 }
 
+#[tokio::test]
+async fn test_struct_literals() -> Result<()> {
+    test_expression!(
+        "STRUCT(1,2,3,4,5)",
+        "{\"f_0\": 1, \"f_1\": 2, \"f_2\": 3, \"f_3\": 4, \"f_4\": 5}"
```

Comment:
I would prefer the generated column names to be `c0`, `c1`, ... but that is just personal preference (and probably based on experience with Spark).

Reply:
Agree! align with Spark.